src/skeleton/Skeleton.java

```diff
@@ -28,5 +30,16 @@ public static void SkeletonCase1() {
 		player.moveTo("jobbra", new Switch());
 		player.shootPortalThroughDoor(new Door(), "jobbra", new SpecialWall());
 	}
+	
+	//Doboz felv�tele, Switchre helyez�se, majd �thalad�s a kiny�lt ajt�n
+	private static void SkeletonCase6() {
+		Player player = new Player();
+		player.moveTo("balra", new Cell());
+		player.moveTo("lefele", new NormalFloor());
```

Comment:
I think we should replace Cells with NormalFloors everywhere, and implement another skeleton method for picking up boxes. Interact is called every time a player moves to an interactable object, picking up the box is meant to be an optional action for the player.